src/selectors/entities/groups.js

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {createSelector} from 'reselect';
+
+export function getAllGroups(state) {
+    return state.entities.groups.groups;
+}
+
+export function getGroup(state, id) {
+    return getAllGroups(state)[id];
+}
+
+export function getGroupMemberCount(state, id) {
+    const memberData = state.entities.groups.members;
+    const groupMemberData = memberData[id];
+    if (!groupMemberData) {
+        return 0;
+    }
+    return memberData[id].totalMemberCount;
+}
+
+const getGroupSyncables = createSelector(
```

Comment:
`getGroupSyncables`, `getGroupTeams`, `getGroupChannels`, and `getGroupMembers` should be factory functions that return selectors, otherwise they'll overwrite each others' memoization. Either that or we can just remove the usage of `createSelector` since they're returning primitives without doing much work.

Reply:
Fixed, PTAL